retrofit/src/main/java/retrofit2/Retrofit.java

```diff
@@ -282,6 +285,27 @@ public BaseUrl baseUrl() {
   }
 
   /**
+   * Returns a {@link Converter} for {@code type} to {@link RequestBody} from the available
+   * {@linkplain #converterFactories() factories}.
+   *
+   * @throws IllegalArgumentException if no converter available for {@code type}.
+   */
+  public <T> Converter<T, RequestBody> requestBodyConverter(Type type, Annotation[] annotations) {
+    return nextRequestBodyConverter(null, type, annotations);
+  }
+
+  /**
+   * Returns a {@link Converter} for {@code type} to {@link RequestBody} from the available
+   * {@linkplain #converterFactories() factories} except {@code skipPast}.
+   *
+   * @throws IllegalArgumentException if no converter available for {@code type}.
+   */
+  public <T> Converter<T, RequestBody> nextRequestBodyConverter(Converter.Factory skipPast,
+      Type type, Annotation[] annotations) {
+    return nextRequestBodyConverter(skipPast, type, annotations, new Annotation[0]);
```

Comment:
We'd probably wanna extract this into a constant.

`static final Annotation[] EMPTY_ANNOTATION_ARRAY = new Annotation[0];`